Cython/Compiler/Tests/TestCmdLine.py

```diff
@@ -12,7 +16,14 @@
 
 from .Utils import backup_Options, restore_Options, check_global_options
 
+unpatched_exists = os.path.exists
 
+def patched_exists(path):
+    if path in ('source.pyx', 'file.pyx', 'file1.pyx', 'file2.pyx', 'file3.pyx', 'foo.pyx', 'bar.pyx'):
+        return True
+    return unpatched_exists(path)
+
+@patch('os.path.exists', new=Mock(side_effect=patched_exists))
```

Comment:
I think this deserves a quick comment. Just something like "avoid the Cython command raising a file not found error"

Reply:
fixed.